pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/manager/controller_manager_config.go

```diff
@@ -64,14 +64,14 @@ webhook:
 leaderElection:
   leaderElect: true
   resourceName: {{ hashFNV .Repo }}.{{ .Domain }}
-# leaderElectionReleaseOnCancel defines if the leader should step down volume
-# when the Manager ends. This requires the binary to immediately end when the
-# Manager is stopped, otherwise, this setting is unsafe. Setting this significantly
-# speeds up voluntary leader transitions as the new leader don't have to wait
-# LeaseDuration time first.
-# In the default scaffold provided, the program ends immediately after
-# the manager stops, so would be fine to enable this option. However,
-# if you are doing or is intended to do any operation such as perform cleanups
-# after the manager stops then its usage might be unsafe.
-# leaderElectionReleaseOnCancel: true
+  # leaderElectionReleaseOnCancel defines if the leader should step down volume
+  # when the Manager ends. This requires the binary to immediately end when the
+  # Manager is stopped, otherwise, this setting is unsafe. Setting this significantly
+  # speeds up voluntary leader transitions as the new leader doesn't have to wait
+  # the LeaseDuration time first.
+  # In the default scaffold provided, the program ends immediately after
+  # the manager stops, so it would be fine to enable this option. However,
+  # if you are doing, or are intending to do, any operation such as perform cleanups
+  # after the manager stops then its usage might be unsafe.
+  # leaderElectionReleaseOnCancel: true
```

Comment:
My 2 cents is that the added space here might cause problems for other places which are replacing this content.
Can you please let us know why you are adding the space?

Reply:
My motivation was to have this properly aligned so the YAML is valid, even though it is all commented out. Basically, to have `leaderElectionReleaseOnCancel` be under the `leaderElection` section, rather than appearing to be a peer.

I did not find issues with the changes format, but if there are concerns that the risk could outweigh the benefit, I can shift that back.

Reply:
I think it is OK we move forward with this one because it is a comment
The problem with changes like that is more problematic when they are in the optional things since other tools might be looking for the text and replacing it.